ccpp/data/GFS_typedefs.F90

```diff
@@ -1752,7 +1757,7 @@ module GFS_typedefs
     real (kind=kind_phys), pointer      :: drain(:)           => null()  !<
     real (kind=kind_phys), pointer      :: drain_in_m_sm1(:)  => null()  !<
     real (kind=kind_phys), pointer      :: dtdt(:,:)          => null()  !<
-    real (kind=kind_phys), pointer      :: dtdtc(:,:)         => null()  !<
+    real (kind=kind_phys), pointer      :: dtdtnp(:,:)        => null()  !<
```

Comment:
Do you need dtdtnp in the interstitial DDT? I don't think so, you also added it Tbd or another DDT, and only that one has a standard name.

Reply:
I will have to sort this one out. I don't know the answer right now.

Reply:
OK, I made sense of it, and removed the dtdtnps in Interstitial